### PR TITLE
[new ui] Toolbar component

### PR DESCRIPTION
**Pull request by walaura**
Add a `<Toolbar />` component to wrap elements like tabbars

[🌈 Demo 🌈](https://deploy-preview-1203--graphiql-test.netlify.com/storybook/?path=/story/toolbar--toolbar-with-tabs)

## Screenshots

Screen Shot 2020-01-08 at 4 55 52 pm
Screen Shot 2020-01-08 at 4 55 56 pm

### CODE REVIEW

**Comment by acao** [comment]
ah! i see what you've done here. looks very nice

File: packages/graphiql/src/new-components/Toolbar/Tabs.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Tab.propTypes = { active: PropTypes.bool };
+
+const Tabs = ({ tabs, active, onChange }) => {
+  return (
+    <WithDividers>
+      {tabs.map((tab, index) => (
+        <Tab
+          key={index}
+          active={active === index}
+          onClick={() => onChange(index)}>
```
Comment (acao) [inline packages/graphiql/src/new-components/Toolbar/Tabs.js]: eventually we might want a readable ID here, but i think that would be up to the particular reducer action that fires when `onChange` happens, not this business-agnostic component.

File: packages/graphiql/src/new-components/Toolbar/support/WithDividers.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      width: '1px',
+    }}
+  />
+);
+Divider.propTypes = { innerSx: PropTypes.object };
+
+const WithDividers = ({ children, padding = false, ...props }) => {
+  return (
+    <ul
+      data-contains-divider
```
Comment (acao) [inline packages/graphiql/src/new-components/Toolbar/support/WithDividers.js]: I didn't know you could pass attributes this way in JSX?

File: packages/graphiql/src/new-components/List/List.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
-
 import PropTypes from 'prop-types';
 
-const ListRow = ({ children, flex = false, padding = true }) => (
+const ListRow = ({ children, flex = false, padding = false }) => (
```
Comment (acao) [inline packages/graphiql/src/new-components/List/List.js]: i like making `padding` explicit, excellent idea

Comment (walaura) [reply packages/graphiql/src/new-components/List/List.js]: i'm a bit conflicted bc ideally itll always be the children of listrow that have padding but it comes in super handy for storybook bc it's mostly text blocks if that makes sense?

**Comment by walaura** [comment]
hold up why are the tabs listed twice on storybook will have a look

File: packages/graphiql/src/new-components/List/List.js (hunk 3 above)
Comment (acao) [reply packages/graphiql/src/new-components/List/List.js]: yeah that does make sense

**Pull request merged**